### ISSUE TITLE
change google picker button css

### ISSUE THREAD

**Issue by hauw123** [sdoomz/react-google-picker#19]
is it possible to change button style / css and value/text ?

**Comment by WouterNoordhof** [comment]
Yes, the standard button is just a normal button, you can give it any className you want, and make it say what you want.
